application/event_processor/event_processor.py

```diff
@@ -1,75 +1,198 @@
-from json import dumps
+from json import dumps, encoder
 from logging import getLogger
 from os import environ, getenv
+import json
+import sys
+sys.path.append("../")
 
 from aws_lambda_powertools import Tracer
-from boto3 import client
-from change_request import ChangeRequest
+import boto3
```

Comment:
Can this be put back to `from boto3 import client`

Reply:
This is to a larger discussion maybe about module referencing

https://github.com/nhsd-exeter/dos-integration/pull/44#discussion_r763174147